sendclient/upload.py

```diff
@@ -98,10 +101,10 @@ def send_file(service, file, fileName=None, password=None, ignoreVersion=False):
     encMeta = encrypt_metadata(keys, fileName)
 
     print('Uploading "' + fileName + '"')
-    secretUrl, fileId, fileNonce, delete_token = api_upload(service, encData, encMeta, keys)
+    secretUrl, fileId, fileNonce, owner_token = api_upload(service, encData, encMeta, keys)
 
     if password != None:
         print('Setting password')
         set_password(service, keys, secretUrl, fileId, password, fileNonce)
 
-    return secretUrl, delete_token
+    return secretUrl, fileId, owner_token
```

Comment:
I don't think you meant to include `fileId` here? (At least not in this PR) 

If you have a use for send_file also returning the fileId say so and this can be left as-is by updating https://github.com/ehuggett/send-cli/blob/ae93aad40dcf67a4122bba780c0fe8a55cefd209/scripts/send-cli#L54-L56 to expect 3 parameters to be returned.

Reply:
I think it would be convenient to return the fileId in that function (in those cases where the direct module is used instead of the CLI), since it's the uploader of the file is the "proper owner" and should be his/her responsibility to delete it.

I will remove it from the PR as you suggest.

Reply:
Done it